pkg/operator/provisioner/provisioner.go

```diff
@@ -14,20 +14,23 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package operator to manage Kubernetes storage.
-package operator
+// Package provisioner to provision Rook volumes on Kubernetes.
+package provisioner
```

Comment:
to make this cleaner. After the refactor, only operator.go and provisioner.go were under the operator package. So it felt that provisioner should also have its own package under `operator` just like the rest. Other than that, there is no real reason.